src/Microsoft.AspNetCore.Mvc.TagHelpers/CacheTagHelper.cs

```diff
@@ -132,17 +132,38 @@ public override int Order
         public TimeSpan? ExpiresSliding { get; set; }
 
         /// <summary>
-        /// Gets or sets the <see cref="CacheItemPriority"/> policy for the cache entry.
-        /// </summary>
-        [HtmlAttributeName(CachePriorityAttributeName)]
-        public CacheItemPriority? Priority { get; set; }
```

Comment:
Nope, only the properties globally supported by the cache storage implementations should be there. Any other implementation can have custom options. If we want to through an exception is a parameter is set but not supported, any implementation can do that, not the context object.